cmd/mothership/local/options.go

```diff
@@ -80,13 +72,19 @@ func componentsFromStrings(list []string, values []string) ([]*keb.Component, er
 	}
 
 	for _, item := range list {
-		s := strings.Split(item, "@")
-		name := s[0]
+		if strings.HasPrefix(item, "{") {
```

Comment:
why are you checking if this string contains `{`? May it be a JSON? If yes why are you not removing the last JSON symbol?

Reply:
Variable `item` can looks like `{reafter,rafterNs,rafterUrl}`
But someone can pass only one value e.g. `rafter` 
This if check if we get three things(name, namespace and url) or one thing(name). 
In previous version we usevchar `@` but it can be problem in url in future. 

In line 76 we remove first char `{` and last char `}`.

Reply:
yeah... right, I missed it 🤦